app/lib/efile/md/md502_calculator.rb

```diff
@@ -74,7 +74,7 @@ def calculate
         @md502r.calculate # depends on 502SU
         set_line(:MD502_LINE_10A, :calculate_line_10a) # depends on 502R
         set_line(:MD502_LINE_13, :calculate_line_13)
-        if filing_status_mfj?
+        if state_filing_status_mfj?
```

Comment:
do we still want to do the worksheet if they're `:dependent` state_filing_status?